Serverless/Azure Functions/Function set up.md

```diff
@@ -0,0 +1,83 @@
+Add as a new project, through VS Function App template. This creates azure function app that can contain multiple functions that share common configuration such as environment variables, app settings and host.
+All functions will be deployed together under same function-app umbrella and  scaled together.
+
+### Triggers
+
+To trigger function execution, different triggers can be used. Most popular ones are :
+
+* Timer - Allow running a function by a schedule ( Example: run a function at midnight every night). Schedule format and samples can be found [here](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-timer?tabs=csharp#ncrontab-expressions).
+
+```c#
+		[Function("FunctionName")]
+		public void Run([TimerTrigger("0 */5 * * * *")] MyInfo myTimer)
+		{
+		    _logger.LogInformation($"C# Timer trigger function executed at: {DateTime.Now}");
+		    _logger.LogInformation($"Next timer schedule at: {myTimer.ScheduleStatus.Next}");
+		    // your logic
+		}
+```
```

Comment:
You can remove paddings (or margins? I'm referring to the empty spaces before the code 😅 ) on all of these code snippets as well, they are moved to the right without any reason :)
![image](https://user-images.githubusercontent.com/96879277/192260813-aa5997df-f986-49e0-8e3a-18f2a1847dfc.png)